android/app/src/main/java/org/jitsi/meet/AudioModeModule.java

```diff
@@ -0,0 +1,440 @@
+         */
+        private boolean headsetAvailable = false;
+
+        /**
+         * Start monitoring Bluetooth device activity.
+         */
+        public void start() {
+            ReactContext reactContext = getReactApplicationContext();
+            if (reactContext == null) {
+                return;
+            }
+
+            bluetoothRunnable = new Runnable() {
+                @Override
+                public void run() {
+                    if (bluetoothHeadset == null) {
+                        headsetAvailable = false;
+                    } else {
+                        List<BluetoothDevice> devices = bluetoothHeadset.getConnectedDevices();
+                        headsetAvailable = !devices.isEmpty();
+                    }
+                    onAudioDeviceChange();
+                }
+            };
+
+            bluetoothAdapter = BluetoothAdapter.getDefaultAdapter();
+            if (bluetoothAdapter == null) {
+                Log.w(TAG, "Device doesn't support Bluetooth");
+                return;
+            }
+
+            if (!audioManager.isBluetoothScoAvailableOffCall()) {
+                Log.w(TAG, "Bluetooth SCO is not available");
+                return;
+            }
+
+            bluetoothProfileListener = new BluetoothProfile.ServiceListener() {
+                @Override
+                public void onServiceConnected(int profile, BluetoothProfile proxy) {
+                    if (profile == BluetoothProfile.HEADSET) {
+                        bluetoothHeadset = (BluetoothHeadset) proxy;
+                        updateDevices();
+                    }
+                }
+
+                @Override
+                public void onServiceDisconnected(int profile) {
+                    if (profile == BluetoothProfile.HEADSET) {
+                        bluetoothHeadset = null;
+                        updateDevices();
+                    }
+                }
+            };
+
+            bluetoothAdapter.getProfileProxy(reactContext,
+                    bluetoothProfileListener, BluetoothProfile.HEADSET);
+
+            IntentFilter bluetoothFilter = new IntentFilter();
+            bluetoothFilter.addAction(AudioManager.ACTION_SCO_AUDIO_STATE_UPDATED);
+            bluetoothFilter.addAction(BluetoothHeadset.ACTION_CONNECTION_STATE_CHANGED);
+            BroadcastReceiver bluetoothReceiver = new BroadcastReceiver() {
+                @Override
+                public void onReceive(Context context, Intent intent) {
+                    final String action = intent.getAction();
+                    if (action.equals(BluetoothHeadset.ACTION_CONNECTION_STATE_CHANGED)) {
+                        final int state = intent.getIntExtra(BluetoothHeadset.EXTRA_STATE, -99);
+                        switch (state) {
+                            case BluetoothHeadset.STATE_CONNECTED:
+                            case BluetoothHeadset.STATE_DISCONNECTED:
```

Comment:
The adopted coding style aligns `switch` and `case`/`default` at the same indentation level. Please comply with it.